packages/render-fidelity-tools/package.json

```diff
@@ -35,29 +35,25 @@
     "@types/mkdirp": "^1.0.1",
     "@types/pngjs": "^3.4.0",
     "@types/puppeteer": "^3.0.1",
-    "@types/resize-observer-browser": "^0.1.2",
-    "@types/rimraf": "^3.0.0",
-    "@webcomponents/webcomponentsjs": "~2.1.3",
+    "@types/rimraf": "^3.0.1",
```

Comment:
@augmentedsky I did not update puppeteer yet, since sometimes they change their API. I recommend you do so soon and test that nothing breaks.